clouda-inc/store-theme-bisco
# lazy load, searchtitle, footer, mobile header logo

#### What is the purpose of this pull request?
Update Footer links
- Update Image URLs for mobile header logo
- Implement lazy fold on search(brand, dpt,cat,subcat, ft search) and product pages for page load improvements
- Re-position Search Title on search pages to left column
- Added recommended products to product detail pages 


#### What problem is this solving?
Miscellaneous UX bugs and page load


#### How should this be manually tested?
Validate page load on google pagespeed insights before and after
Check on Footer links (ensure they all navigate correctly)
Validate bisco logo header on mobile view
Check search title location on search result pages
#### Screenshots or example usage

#### Types of changes

* [ X] Bug fix (a non-breaking change which fixes an issue)
* [ X] New feature (a non-breaking change which adds functionality)
